docs/dashboard.json

```diff
@@ -0,0 +1,1269 @@
+{
```

Comment:
Does this normally go in the ./docs folder?

Reply:
Lol no, but I'm not sure where to put it. Maybe under `deploy/grafana`?